css-positioning/absolute.css

```diff
@@ -0,0 +1,45 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  right: 20px;
+  top: 5%;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  top: 51%;
```

Comment:
![image](https://user-images.githubusercontent.com/52118958/101266310-78bc4a00-3702-11eb-96a4-8e0dd3a763fe.png)

Reply:
!!! I though I had pushed these corrections too, but when I double-checked:
![image](https://user-images.githubusercontent.com/73370554/101266366-1748ab00-3703-11eb-8339-0b27c1379de8.png)

I will never live down this shame